examples/mining_ind.py

```diff
@@ -0,0 +1,39 @@
+import desbordante
+
+def print_table(filename: str):
+    with open(filename) as table:
+        rows = [row.strip().split(',') for row in table.readlines()]
+    desired_lens = [0 for i in range(len(rows))]
+    for col_num in range(len(rows[0])):
+        max_len = 0
+        for row in rows:
+            max_len = max(max_len, len(row[col_num]))
+        desired_lens[col_num] = max_len + 3
+    for row_num in range(len(rows)):
+        for col_num in range(len(rows[row_num])):
+            item = rows[row_num][col_num]
+            spaces_num = desired_lens[col_num] - len(item)
+            print(f'{item}{" " * spaces_num}', end='')
+        print()
+        if (row_num == 0):
+            print('-' * sum(desired_lens))
```

Comment:
```suggestion
    header, *data_rows = rows
    print(row_to_padded_string(header, column_widths))
    print('-' * sum(column_widths))
    print('\n'.join(row_to_padded_string(row, column_widths) for row in data_rows))
```

```py
def row_to_padded_string(row, widths):
    return ''.join(field.ljust(width) for width, field in zip(widths, row))
```

https://docs.python.org/3/library/stdtypes.html#str.ljust
https://docs.python.org/3/library/functions.html#zip
https://docs.python.org/3/library/stdtypes.html#str.join